### PR TITLE
use exported robotlist and envlist

### PR DESCRIPTION
**Pull request by fmessmer**
as discussed/proposed in https://github.com/ipa320/cob_robots/issues/577

### CODE REVIEW

File: cob_navigation_global/CMakeLists.txt
Comment (mgruhler) [inline cob_navigation_global/CMakeLists.txt]: @ipa-fxm why did you comment out all tests?

File: cob_navigation_local/CMakeLists.txt
Comment (mgruhler) [inline cob_navigation_local/CMakeLists.txt]: as above

File: cob_navigation_slam/CMakeLists.txt
Comment (mgruhler) [inline cob_navigation_slam/CMakeLists.txt]: as above

**Review by mgruhler — CHANGES_REQUESTED** [review]
I don't like that all tests are removed in this repo atm.

File: cob_navigation_slam/CMakeLists.txt
Comment (fmessmer) [reply cob_navigation_slam/CMakeLists.txt]: ah, I started with this package and here the tests were commented out....then copied to the other packages :wink: 
you want me to activate all three or just the ones that have been active?

Comment (mgruhler) [reply cob_navigation_slam/CMakeLists.txt]: fair enough :laughing: 

I'd say activate all three.

~~However, thinking about this, we should probably make sure to also remove the now unsupported robots according to your work in cob_robots.~~ (I've just seen #77 :rofl:)  Also, I'm not sure if we have all environments fully set-up in cob_environments. But activating the tests will force us to do this :grin:

Comment (fmessmer) [reply cob_navigation_slam/CMakeLists.txt]: well, yes, this PR is rather meant as a reminder to add this feature to cob_navigation in order to close https://github.com/ipa320/cob_robots/issues/577 rather than merging this **before** #77 
I'd suggest to merge this one **after** #77

Comment (mgruhler) [reply cob_navigation_slam/CMakeLists.txt]: I figured as much by now

Comment (fmessmer) [reply cob_navigation_slam/CMakeLists.txt]: tests added again...still wait for #77

**Comment by fmessmer** [comment]
rebased after #77 was merged

**Comment by fmessmer** [comment]
superseded by #83

**Pull request closed without merging**
